docs/proposal/tls_cert_zh.md

```diff
@@ -0,0 +1,130 @@
+## 证书申请与管理模块
+
+## 概要
+
+kmesh支持TLS能力需要使用由istiod签发的证书，所以需要一套证书申请与管理模块，用于申请证书并管理证书的生命周期。
+
+## 功能
+
+1. 为纳管pod所在的sa申请证书
+2. 证书有效期到期自动刷新
+
+## 证书申请模块
+
+随kmesh启动创建一个caclient客户端，与istiod建立加密的grpc连接
+
+使用workload中的信息，构造出CSR请求和私钥，将CSR请求通过caclient发送给istiod，istiod进行签名并返回证书
+
+## 证书生命周期管理
+
+### 设计细节
+
+**使用一个pending优先队列和map来记录与管理证书**
+
+penging优先队列：使用证书到期时间进行排序，保持最近到期的证书的优先度最高，定期检查最近到期的证书，提前10分钟刷新证书；
+
+```go
+优先队列元素内容：
+type certExp struct {
+    hostName string	//使用sa构造的证书名
+    exp time.Time	//证书到期时间
+}
+
+更新时机：
+	新增证书：插入一条新的记录
+	刷新证书：删除旧记录，添加新记录；
+	删除证书：遍历并删除旧证书的记录
+```
+
+使用map来记录证书信息和证书状态：
+
+```go
+map：记录使用该证书的pod 数量
+​	key：hostName	//使用sa构造的证书名
+​	value：certCache
+
+type certCache struct {
+	cert istiosecurity.SecretItem	//证书信息
+    refcnt uint32 //记录使用该证书的pod数
+    refIp uint64  //记录使用该证书的workload.addr

```

Comment:
What is the correspondence designed here? The `refcnt` field records the number of pods that use the certificate. It seems that a certificate is used by multiple pods. Does the certificate correspond to multiple workloads? Why is `refIp` not an array?

Reply:
Clerical error. refIp will be deprecated. An SA certificate corresponds to all workloads that use the SA.